rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -299,6 +302,96 @@ void gs_frame::flip(draw_context_t, bool /*skip_frame*/)
 	}
 }
 
+void gs_frame::take_screenshot(std::vector<u8> ss_data, const u32 ss_width, const u32 ss_height, bool vulkan_buf)
+{
+	std::thread(
+	    [ss_width, ss_height, vulkan_buf](std::vector<u8> sshot_data) {
+		    std::string screen_path = fmt::format("%s/Screenshots/", fs::get_config_dir());
+
+		    if (!fs::is_dir(screen_path))
+		    {
+			    if (!fs::create_dir(screen_path))
+			    {
+				    LOG_ERROR(GENERAL, "Failed to create screenshot path: %s", screen_path);
+				    return;
+			    }
+		    }
+
+		    u32 file_index;
+		    for (file_index = 0; file_index < 100000000; file_index++)
```

Comment:
```suggestion
		    for (file_index = 1; file_index < 100'000'000; file_index++)
```
Starting indexing at 1 would feel more natural to non-programmers.

Reply:
Maybe use wall clock instead for name? this way it would be easier to distinguish one screenshot from another by name.

Reply:
and it doesn't have "out of screenshots places" problem because you cant repeat the same date+time twice.

Reply:
Yes using date and time makes sense, changed it.